### PR TITLE
Delete Autocrashic  :'D

### PR DESCRIPTION
**Pull request by Spandamn**
(No metabashing intended)
April Fools, enjoy the omotms

### CODE REVIEW

**Comment by Spandamn** [comment]
Forgot to tag @TheImmortal

File: config/formats.js [hunk 1]
```diff
@@ -286,45 +286,43 @@ exports.Formats = [
...
 		],
-
-		mod: 'fullpotential',
-		ruleset: ['[Gen 7] OU', 'Item Clause'],
-		banlist: ['Dugtrio-Base', 'Pheromosa', 'Raichu-Alola', 'Shuckle', 'Chlorophyll', 'Sand Rush', 'Slush Rush', 'Speed Boost', 'Swift Swim', 'Unburden'],
+		mod: 'gen7',
+		ruleset: ['[Gen 7] OU'],
+		onModifyTemplate: function (template, pokemon) {
+			let types = [this.getMove(pokemon.moves[0]).type];
+			if (pokemon.moves[1] && this.getMove(pokemon.moves[1]).type !== types[0]) types.push(this.getMove(pokemon.moves[1]).type);
```
Comment (urkerab) [inline config/formats.js]: This doesn't handle Hidden Power correctly, does it? My understanding is that OP wants it to be Normal type.

File: config/formats.js [hunk 2]
```diff
@@ -286,45 +286,43 @@ exports.Formats = [
...
+			template.types = types;
```
Comment (urkerab) [inline config/formats.js]: Please don't change the existing template object.

File: config/formats.js [hunk 3]
```diff
@@ -294,10 +294,14 @@ exports.Formats = [
...
-			let types = [this.getMove(pokemon.moves[0]).type];
-			if (pokemon.moves[1] && this.getMove(pokemon.moves[1]).type !== types[0]) types.push(this.getMove(pokemon.moves[1]).type);
-			template.types = types;
-			return template;
+			let temp = Object.assign({}, template);
+			let types = [pokemon.moves[0] === 'hiddenpower' ? pokemon.hpType : this.getMove(pokemon.moves[0]).type];
+			if (pokemon.moves[1]) {
+				let move2type = pokemon.moves[1] === 'hiddenpower' ? pokemon.hpType : this.getMove(pokemon.moves[1]).type;
+				if (move2type !== types[0]) types.push(move2type);
+			}
```
Comment (urkerab) [inline config/formats.js]: What part of 'OP wants Hidden Power to count as Normal type' did you not understand...

Comment (TheImmortal) [inline config/formats.js]: Wait what. That doesn't make any sense. Hidden Type should not be affected by the mechanic of this OM.

Comment (Spandamn) [inline config/formats.js]: Tfw i read that as doesn't

**Pull request merged**